site/docs/patterns/preferences-dialog.mdx

```diff
@@ -0,0 +1,178 @@
+
+The visual indicators are the two separator lines above and below the parent-child layout. Apply the shadow-scroll token for the container shadow. 
+
+  <Diagram
+    src="/img/patterns/preferences-dialog/scrolling.png"
+    alt="Example of scrolling."
+    border={false}
+  />
+
+You should display scrolling affordance when the content or navigation (or both) exceeds the visible area of the parent-child layout.
+
+  <Diagram
+    src="/img/patterns/preferences-dialog/scrolling-affordance.png"
+    alt="Example of scrolling affordance."
+    border={false}
+  />
+
+## Applying preferences
+
+**Cancel** and **Save** button bar actions allow the user to manually apply preferences in the application. We recommend this when the user wants to:
+
+- Make multiple changes with ease and apply them simultaneously.
+- Easily cancel any unwanted changes.
+
+It's also preferable if it’s technically complex or taxing on performance for affected components in your application to listen for live changes in the preferences dialog and dynamically update themselves.
+
+<Diagram
+  src="/img/patterns/preferences-dialog/applying-preferences.png"
+  alt="Applying preferences."
+  border={false}
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+  Display core actions to the right of a dialog. Refer to the button bar pattern for further guidance on how to display user actions within dialogs. 
+</ExampleContainer>
+
+### Live updates
+
+If your application can listen for changes and take immediate effect, you don't need to display a button bar across a preferences dialog. A close button in the top-right corner of the dialog should be on display instead to allow the user to close the dialog after editing any preferences.
+
+<Diagram
+  src="/img/patterns/preferences-dialog/live-updates.png"
+  alt="Live updates."
+  border={false}
+/>
+
+## Secondary actions
+
+We recommend configuring the button bar to display secondary actions to the left such as **Import** and **Export** if you need to enable additional actions.
+
+<Diagram
+  src="/img/patterns/preferences-dialog/secondary-actions.png"
+  alt="Live updates."
+  border={false}
+/>
+
+## Responsiveness
+
+The preferences dialog is responsive to the width of the application viewport and respond dynamically to the Salt breakpoints. 
+
+Once a preference dialog hits a certain breakpoint, you can configure the layout to either: 
+
+1. Collapse to allow more room for content.
+2. Change the orientation to stack the button bar.
+
+### Collapsed navigation
+
+When the navigation collapses, the preferences content displays across the entire width of the dialog. To select and view alternative preference types, users should use the back chevron button to navigate back to the vertical navigation. The vertical navigation view also spans the dialog width. A collapsed layout ensures improved usability of dialog content in smaller screen sizes or application resolutions.
```

Comment:
We switch between "displays across" and "spans" in the same sentence. Should we just use "spans"

Reply:
```suggestion
When the navigation collapses, the preferences content spans the entire width of the dialog. To select and view alternative preference types, users should use the back chevron button to navigate back to the vertical navigation. The vertical navigation view also spans the dialog width. A collapsed layout ensures improved usability of dialog content in smaller screen sizes or application resolutions.
```